tools/ide_plugins/pysa-vscode/syntaxes/Pysa.tmLanguage

```diff
@@ -0,0 +1,4007 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
A few requests here:
1. Can you provide the source so we can check licensing?
2. Can you point out which lines you changed? Maybe the output of `diff` or something? It'll be too cumbersome to review a whole fork of a 3k line file.

Reply:
Yep, for sure - I've made a gist [here](https://gist.github.com/gracewgao/f816b7c27ad66b76df9961e3e43a0d19/revisions) to compare the differences. To summarize, I basically just changed the scope names to be `pysa` rather than `python`, fixed the UUID, and made all the other changes I mentioned in the PR description.